DeviceTests/DeviceTests.Shared/Launcher_Tests.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Xamarin.Essentials;
+using Xunit;
+
+namespace DeviceTests
+{
+    // TEST NOTES:
+    //   - a human needs to close the browser window
+    public class Launcher_Tests
+    {
+        [Theory]
+        [InlineData("http://www.example.com")]
```

Comment:
Some thoughts on more inline data tests:

```
https://www.example.com
http://example.com/?query=blah
https://example.com/?query=blah
mailto://someone@microsoft.com
mailto://someone@microsoft.com?subject=test
```

Reply:
Should be good now.